engine/crates/parser-sdl/src/rules/basic_type.rs

```diff
@@ -29,15 +33,24 @@ impl<'a> Visitor<'a> for BasicType {
         type_definition: &'a engine::Positioned<engine_parser::types::TypeDefinition>,
     ) {
         let directives = &type_definition.node.directives;
-        if_chain! {
```

Comment:
Got rid of this `if_chain` stuff because it's quite annoying - breaks rust analyzer and we have `let else` as an alternative now anyway

Reply:
100% agree